[json-serialize-action] Unsupported field: unmarshalTypeName

When using the _json-serialize-action_ action with camel-jbang (camel 3.14.0) with jbang, I get:

```
2021-12-22 10:13:58.628  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  : Auto-configuration summary
2021-12-22 10:13:58.630  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  :     camel.main.name=CamelJBang
2021-12-22 10:13:58.630  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  :     camel.main.shutdownTimeout=5
2021-12-22 10:13:58.630  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  :     camel.main.routesReloadEnabled=false
2021-12-22 10:13:58.630  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  :     camel.main.routesIncludePattern=file:r.yaml
2021-12-22 10:13:58.631  INFO 45033 --- [           main] o.a.c.m.BaseMainSupport                  :     camel.component.kamelet.location=classpath:/kamelets,github:apache:camel-kamelets/kamelets
2021-12-22 10:13:59.162  INFO 45033 --- [           main] o.a.c.m.DownloaderHelper                 : Downloaded dependency: org.apache.camel.kamelets:camel-kamelets-utils:0.5.0 took: 440ms
2021-12-22 10:13:59.171  WARN 45033 --- [           main] o.a.c.i.e.AbstractCamelContext           : Lifecycle strategy org.apache.camel.component.kamelet.KameletComponent$LifecycleHandler@1a4ae53d vetoed initializing CamelContext (CamelJBang) due to: Failure creating route from template: json-serialize-action
2021-12-22 10:13:59.171  INFO 45033 --- [           main] o.a.c.i.e.AbstractCamelContext           : CamelContext (CamelJBang) vetoed to not initialize due to Failure creating route from template: json-serialize-action
org.apache.camel.RuntimeCamelException: org.apache.camel.VetoCamelContextStartException: Failure creating route from template: json-serialize-action
	at org.apache.camel.RuntimeCamelException.wrapRuntimeException(RuntimeCamelException.java:66)
	at org.apache.camel.support.service.BaseService.doFail(BaseService.java:413)
	at org.apache.camel.impl.engine.AbstractCamelContext.doFail(AbstractCamelContext.java:3482)
	at org.apache.camel.support.service.BaseService.fail(BaseService.java:342)
	at org.apache.camel.impl.engine.AbstractCamelContext.failOnStartup(AbstractCamelContext.java:5026)
	at org.apache.camel.impl.engine.AbstractCamelContext.init(AbstractCamelContext.java:2579)
	at org.apache.camel.support.service.BaseService.start(BaseService.java:111)
	at org.apache.camel.impl.engine.AbstractCamelContext.start(AbstractCamelContext.java:2586)
	at org.apache.camel.impl.DefaultCamelContext.start(DefaultCamelContext.java:247)
	at org.apache.camel.main.KameletMain.doStart(KameletMain.java:128)
	at org.apache.camel.support.service.BaseService.start(BaseService.java:119)
	at org.apache.camel.dsl.jbang.core.commands.Run.run(Run.java:233)
	at org.apache.camel.dsl.jbang.core.commands.Run.call(Run.java:94)
	at org.apache.camel.dsl.jbang.core.commands.Run.call(Run.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at org.apache.camel.dsl.jbang.core.commands.CamelJBangMain.run(CamelJBangMain.java:41)
	at main.CamelJBang.main(CamelJBang.java:35)
Caused by: org.apache.camel.VetoCamelContextStartException: Failure creating route from template: json-serialize-action
	at org.apache.camel.component.kamelet.KameletComponent$LifecycleHandler.onContextInitialized(KameletComponent.java:422)
	at org.apache.camel.impl.engine.AbstractCamelContext.doInit(AbstractCamelContext.java:2870)
	at org.apache.camel.support.service.BaseService.init(BaseService.java:83)
	at org.apache.camel.impl.engine.AbstractCamelContext.init(AbstractCamelContext.java:2567)
	... 18 more
Caused by: java.lang.IllegalArgumentException: Unsupported field: unmarshalTypeName on org.apache.camel.model.dataformat.JsonDataFormat
```

The route is:

```yaml
- from:
    uri: "kamelet:timer-source"
    parameters:
      period: "1s"
      message: '{ "msg": "Hello Json" }'
    steps:
    - to: "kamelet:json-serialize-action"
    - to: "log:info"    
```

The command to run it is:

```
jbang -Dcamel.jbang.version=3.14.0 CamelJBang@apache/camel run my-route.yaml
```

--- a/kamelets/avro-deserialize-action.kamelet.yaml
+++ b/kamelets/avro-deserialize-action.kamelet.yaml
@@ -65,7 +65,7 @@ spec:
       - unmarshal:
           avro:
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
             schemaResolver: "#class:org.apache.camel.kamelets.utils.serialization.InflightAvroSchemaResolver"
       - remove-property:
           name: schema

--- a/kamelets/avro-serialize-action.kamelet.yaml
+++ b/kamelets/avro-serialize-action.kamelet.yaml
@@ -65,7 +65,7 @@ spec:
       - marshal:
           avro:
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
             schemaResolver: "#class:org.apache.camel.kamelets.utils.serialization.InflightAvroSchemaResolver"
       - remove-property:
           name: schema

--- a/kamelets/extract-field-action.kamelet.yaml
+++ b/kamelets/extract-field-action.kamelet.yaml
@@ -57,7 +57,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "field"
           constant: "{{field}}"
@@ -69,7 +69,7 @@ spec:
             - marshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
             - set-header:
                 name: "Content-Type"
                 constant: "application/json"

--- a/kamelets/hoist-field-action.kamelet.yaml
+++ b/kamelets/hoist-field-action.kamelet.yaml
@@ -54,7 +54,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "field"
           constant: "{{field}}"
@@ -62,7 +62,7 @@ spec:
       - marshal:
           json: 
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-header:
           name: "Content-Type"
           constant: "application/json"

--- a/kamelets/insert-field-action.kamelet.yaml
+++ b/kamelets/insert-field-action.kamelet.yaml
@@ -69,7 +69,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "field"
           constant: "{{field}}"
@@ -84,7 +84,7 @@ spec:
             - marshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
             - set-header:
                 name: "Content-Type"
                 constant: "application/json"

--- a/kamelets/json-deserialize-action.kamelet.yaml
+++ b/kamelets/json-deserialize-action.kamelet.yaml
@@ -43,6 +43,6 @@ spec:
       - unmarshal:
           json: 
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - remove-header:
           name: "Content-Type"

--- a/kamelets/json-serialize-action.kamelet.yaml
+++ b/kamelets/json-serialize-action.kamelet.yaml
@@ -43,7 +43,7 @@ spec:
       - marshal:
           json: 
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-header:
           name: "Content-Type"
           constant: "application/json"

--- a/kamelets/mask-field-action.kamelet.yaml
+++ b/kamelets/mask-field-action.kamelet.yaml
@@ -62,7 +62,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "fields"
           constant: "{{fields}}"
@@ -77,7 +77,7 @@ spec:
             - marshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
             - set-header:
                 name: "Content-Type"
                 constant: "application/json"

--- a/kamelets/message-timestamp-router-action.kamelet.yaml
+++ b/kamelets/message-timestamp-router-action.kamelet.yaml
@@ -69,7 +69,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "topicFormat"
           constant: "{{topicFormat}}"
@@ -86,7 +86,7 @@ spec:
       - marshal:
           json: 
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-header:
           name: "Content-Type"
           constant: "application/json"

--- a/kamelets/openai-classification-action.kamelet.yaml
+++ b/kamelets/openai-classification-action.kamelet.yaml
@@ -97,7 +97,7 @@ spec:
       - unmarshal:
           json: 
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - choice:
           when:
           - simple: "${body.has('model')} == false"

--- a/kamelets/protobuf-deserialize-action.kamelet.yaml
+++ b/kamelets/protobuf-deserialize-action.kamelet.yaml
@@ -54,7 +54,7 @@ spec:
       - unmarshal:
           protobuf:
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
             schemaResolver: "#class:org.apache.camel.kamelets.utils.serialization.InflightProtobufSchemaResolver"
       - remove-property:
           name: schema

--- a/kamelets/protobuf-serialize-action.kamelet.yaml
+++ b/kamelets/protobuf-serialize-action.kamelet.yaml
@@ -54,7 +54,7 @@ spec:
       - marshal:
           protobuf:
             library: Jackson
-            unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+            unmarshalType: com.fasterxml.jackson.databind.JsonNode
             schemaResolver: "#class:org.apache.camel.kamelets.utils.serialization.InflightProtobufSchemaResolver"
       - remove-property:
           name: schema

--- a/kamelets/replace-field-action.kamelet.yaml
+++ b/kamelets/replace-field-action.kamelet.yaml
@@ -68,7 +68,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "enabled"
           constant: "{{enabled}}"
@@ -86,7 +86,7 @@ spec:
             - marshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
             - set-header:
                 name: "Content-Type"
                 constant: "application/json"

--- a/kamelets/value-to-key-action.kamelet.yaml
+++ b/kamelets/value-to-key-action.kamelet.yaml
@@ -57,7 +57,7 @@ spec:
             - unmarshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
       - set-property:
           name: "fields"
           constant: "{{fields}}"
@@ -69,7 +69,7 @@ spec:
             - marshal:
                 json: 
                   library: Jackson
-                  unmarshalTypeName: com.fasterxml.jackson.databind.JsonNode
+                  unmarshalType: com.fasterxml.jackson.databind.JsonNode
             - set-header:
                 name: "Content-Type"
                 constant: "application/json"

